src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-list/rbd-list.component.ts

```diff
@@ -307,6 +312,10 @@ export class RbdListComponent extends ListWithDetails implements OnInit {
       itemFilter,
       this.builders
     );
+
+    this.rbdPoolSubscription = this.rbdService.list().subscribe((resp: Array<RbdPool>) => {
```

Comment:
The subscription to this service is already done [here](https://github.com/ceph/ceph/blob/fd3a9eb92f807e8f4571c2436c431595e41e2dd4/src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-list/rbd-list.component.ts#L306)  and it's getting processed in the next line, 
`      (resp) => this.prepareResponse(resp),`
So you can probably add a extra line inside function `this.rbdPools = resp` or I think a boolean  is enough too.
`rbdPoolsExist = !(resp === undefined || resp.length == 0)`